pandas/util/_print_versions.py

```diff
@@ -33,7 +33,7 @@ def _get_commit_hash() -> str | None:
     except ImportError:
         from pandas._version import get_versions
 
-        versions = get_versions()
+        versions = get_versions()  # type: ignore[no-untyped-call]
```

Comment:
Could annotate `get_versions` in `pandas/_version.py` to return `dict[str, Any]`

but I believe we try to avoid changing copies of external files @mroeschke